### PR TITLE
Add checks and errors for common problems in start script

### PR DESCRIPTION
**Pull request by tzach**
This PR add proper error messages to problems in start-all.sh script:
- docker engine is not running
- Prometheus failed to start
- Grafana failed to start

### CODE REVIEW

**Comment by tzach** [comment]
@pdziepak @tgrabiec please review

File: start-all.sh [hunk 1]
```diff
@@ -20,6 +20,13 @@ while getopts ':hd:' option; do
   esac
 done
 
+# Exit if Docker engine is not running
+if [ ! "$(sudo docker ps)" ]
+then
+        echo "Error: Docker engine is not running"
+        exit 1;
```
Comment (amnonh) [inline start-all.sh]: no need for the semi colon

File: start-all.sh [hunk 2]
```diff
@@ -28,19 +35,46 @@ else
...
+until $(curl --output /dev/null -f --silent http://localhost:9090) || [ $TRIES -eq $RETRIES ]; do
+    printf '.'
+    ((TRIES=TRIES+1))
+    sleep 5
+done
+
+if [ ! "$(sudo docker ps -q -f name=aprom)" ]
+then
+        echo "Error: Prometheus container failed to start"
+        exit 1;
```
Comment (amnonh) [inline start-all.sh]: no need for the semi colon

File: start-all.sh [hunk 3]
```diff
@@ -28,19 +35,46 @@ else
...
+until $(curl --output /dev/null -f --silent http://localhost:3000/api/org) || [ $TRIES -eq $RETRIES ] ; do
     printf '.'
+    ((TRIES=TRIES+1))
     sleep 5
 done
 
+if [ ! "$(sudo docker ps -q -f name=agraf)" ]
+then
+        echo "Error: Grafana container failed to start"
+        exit 1;
```
Comment (amnonh) [inline start-all.sh]: here to

**Comment by tzach** [comment]
Thanks @amnonh please review the latest version, including your test and a UX improvement

**Comment by tzach** [comment]
Tested on Ubuntu 16, Ubuntu 14 and Fedora 24

**Pull request merged**